### ISSUE TITLE
​Implementation Question

### ISSUE THREAD

**Issue by 1439238771** [GaozhengPei/QEDG#2]
Author, I noticed that the training data directly uses the samples and labels from the memory bank in the implementation. Doesn’t this simplify the clone model’s training process (by augmenting samples from the memory bank and then using the target labels to train the clone model)? If so, where is the loss implementation for non-disputed and disputed samples in the clone model training, as described in the paper?

**Comment by GaozhengPei** [comment]
Sorry for the late response to the issue—the code has already been updated.
